tests/tabular_46.t

```diff
@@ -2,7 +2,7 @@
 . bash_tap_fd.sh
 
 PASSWD=$(get_password)
-CMD="$(get_script) --tabular"
+CMD="$(get_script) --format tabular"
```

Comment:
Keep a copy of the `--tabular` test for backwards compat. testing. Call it `tabular_deprecated_46.t`
The new test looks good.